### PR TITLE
GH actions (EXPOSUREAPP-13085)

### PR DESCRIPTION
**Pull request by mtwalli**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13085)

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/playbook/OrganizerPlaybook.kt [hunk 1]
```diff
@@ -175,13 +174,8 @@ class OrganizerPlaybook @Inject constructor(
             ErrorType.TAN -> OrganizerSubmissionException.ErrorCode.TAN_OB_CLIENT_ERROR
             ErrorType.SUBMISSION -> OrganizerSubmissionException.ErrorCode.SUBMISSION_OB_CLIENT_ERROR
         }
-        // HTTP status code 5XX
-        is CwaServerError -> when (type) {
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/playbook/OrganizerPlaybook.kt:179]: Why is this removed?

**Review by chiljamgossow — COMMENTED** [review]
Nice clean up!

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/playbook/OrganizerPlaybook.kt (hunk 1 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/playbook/OrganizerPlaybook.kt:179]: it is the same as `else` , it was duplicate

**Review by schauersbergern — APPROVED** [review]
The technical debt subtasks could have their own pull requests. Besides of that it looks fine

**Review by chiljamgossow — APPROVED** [review]
